actions/bump-version/action.yaml

```diff
@@ -28,8 +28,8 @@ runs:
     - name: Bump version
       id: release-tag
       run: |
-        echo "oldTag=$(python ./actions/bump-version/scripts/get_version.py)" >> $GITHUB_OUTPUT
+        echo "oldTag=$(python -c "from configparser import ConfigParser; cfg = ConfigParser(); cfg.read('.bumpversion.cfg'); print(cfg['bumpversion']['current_version'])")" >> "$GITHUB_OUTPUT"
         bump2version ${{ inputs.release-type }} --no-commit
-        echo "newTag=$(python ./actions/bump-version/scripts/get_version.py)" >> $GITHUB_OUTPUT
+        echo "newTag=$(python -c "from configparser import ConfigParser; cfg = ConfigParser(); cfg.read('.bumpversion.cfg'); print(cfg['bumpversion']['current_version'])")" >> "$GITHUB_OUTPUT"
```

Comment:
According to this link https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
The `$GITHUB_OUTPUT` is not in quotes.

Reply:
probably actionlint would complain about potential word splitting if we remove the double quotes 😄
Probably it's fine either way

Reply:
For me it is also fine with the double quotes since we tested that it works